index.html

```diff
@@ -550,10 +550,17 @@ <h3 id="placement-of-group-ruby">Placement of group-ruby</h3>
     are treated as a unit, 
     so there is no line wrapping opportunity inside either string.</p>
 	
-<aside class="note" title="Wrap opportunities in group-ruby" id="n20200529006">
+<aside class="note" title="Line wrapping in group-ruby" id="n20200529006">
 <p>As group-ruby is treated as a single unit, there is no wrap opportunity.
       However, there are examples where allowing wrapping may be desirable.
       In such cases, based on appropriate association of base text and ruby annotations, it may be appropriate to handle the wrapping opportunities in the same way as for jukugo-ruby.</p>
+<p>As <a title="group-ruby">group-ruby</a> is treated as a single unit, there is no wrap opportunity. 
```

Comment:
This sentence is the same as the first sentence of the previous paragraph. Is this intended?

Reply:
forgot to remove the first paragraph. thx for pointing.